code/day2/5_actel_custom_spatial.R

```diff
@@ -0,0 +1,89 @@
+## Actel Part 2 - Custom Spatial Networks:
+
+# Let's design a spatial.txt file for our detection data
+
+# Let's first visualize our study area, with a popup that tells us what 
+# project each deployment belongs to:
+
+# Designing a spatial.txt file -----
+
+library(mapview)
+library(spdplyr)
```

Comment:
had to run `install.packages("spdplyr")` should check setup.md to ensure its there

Reply:
this is in setup.R i just seemed to have missed it